client/src/dbus/item.rs

```diff
@@ -195,4 +196,9 @@ impl<'a> Item<'a> {
             Ok(())
         }
     }
+
+    /// Returns item path
+    pub fn path(&self) -> &ObjectPath<'_> {
+        self.inner.inner().path()
+    }
```

Comment:
If we are exposing the path in the dbus::api::Item, I would expect the same thing to be done collections as well. Or maybe just making it possible to compare an `Item<'_>` and `OwnedObjectPath` without exposing the internals as API.

Reply:
I'm not sure if obscuring the compare operation is a great idea. I'm also not sure if it would work properly since neither Eq nor ObjectPath are defined in oo7.

About Collection: The Item path is added in the MR since it also gets exposed elsewhere with API introduced in this MR. Therefore I would suggest moving the discussion for Collection elsewhere since the Collection path doesn't appear in any API yet.